### DISCUSSION TITLE
List all tests of a group

### DISCUSSION THREAD

**Discussion by pixelbrackets** [Codeception/Codeception#6200] [Ideas]
#### What are you trying to achieve?

It is possible to execute one or several groups of tests in Codeception. There are multiple ways to assign a test to a group (tag with `@group` annotations, add a group configuration in a `yaml` or add a group `txt` file).

Is there a way to list all tests which are assigned to a specific group? Eg. “List all tests which are assigned to group »slow«”.

#### What do you get instead?

I tried `codecept dry-run`, but this does not accept the `group` parameter. And `codecept run` executes all tests, which I dont want. I only want to list the tests, like a search result.

### Details

* Codeception version: 2.5.6 (also tested with 3.1.2)
* PHP Version: 7.3
* Operating System: Ubuntu
* Installation type: Composer

**Comment by Naktibalda** [comment]
No, there is no way to do that, but it would make sense to implement `group` parameter for `dry-run` command.

**Comment by cebe** [comment]
The same question has been posted on Stackoverflow. Here is my answer: https://stackoverflow.com/a/74446041/1106908
